tests/acceptance/features/bootstrap/CliContext.php

```diff
@@ -118,6 +120,25 @@ public function theAdministratorRemovesAllVersionsOfResources() {
 		$this->featureContext->setResponse(CliHelper::runCommand($body));
 	}
 
+	/**
+	 * @When the administrator removes the versions of the file :file of user :user in space :space using file-id using CLI
+	 *
+	 * @param string $file
+	 * @param string $user
+	 * @param string $space
+	 *
+	 * @return void
+	 */
+	public function theAdministratorRemovesTheVersionsOfFileUsingFileId($file, $user, $space) {
+		$path = $this->featureContext->getStorageUsersRoot();
+		$fileId = $this->spacesContext->getFileId($user, $space, $file);
+		$command = "revisions purge -p $path --r $fileId --dry-run=false";
```

Comment:
```suggestion
		$command = "revisions purge -p $path -r $fileId --dry-run=false";
```

Reply:
@saw-jan both --r and -r flag works in the command. In the README.MD, it says either --resource-id or --r.
What should we keep for the command?

Reply:
usually, the format would be `--<long-form>` and `-<short-form>`

Reply:
okay, I am gonna keep -r flag and update it.